src/backend/cdb/cdbcat.c

```diff
@@ -534,3 +536,15 @@ checkPolicyForUniqueIndex(Relation rel, AttrNumber *indattr, int nidxatts,
 		}
 	}
 }
+
+void 
+GpPolicyReplace(Oid tbloid, const GpPolicy *policy)
+{
+	GpPolicyReplace_inner(tbloid, policy, false);
+}
+void 
+GpPolicyReplaceWithBucketNum(Oid tbloid, const GpPolicy *policy)
```

Comment:
The older `GpPolicyReplace` function doesn't update `bucketnum` field though I have already set this field in `policy` structure, because `repl[1] = false;`
So I add a new GpPolicyReplaceWithBucketNum function to do it.